database.py

```diff
@@ -109,10 +116,11 @@ def connect_and_initialize():
 
 def _create_citationdetective_tables(cfg, cursor):
     cursor.execute('''
-        CREATE TABLE IF NOT EXISTS statements (id VARCHAR(128) PRIMARY KEY, 
+        CREATE TABLE IF NOT EXISTS statements (
+        id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY, hashid VARCHAR(128),
```

Comment:
Why do we need `hashid` if we also have an auto-incrementing id?

Reply:
Yeah we don't need it. I leave hashid because at first I used it to check repeat sentences in the database. I'll remove it later.